### ISSUE TITLE
GitHub installation issue

### ISSUE THREAD

**Issue by IndrajeetPatil** [easystats/modelbased#107]
I see:

```r
> remotes::install_github("easystats/modelbased")
Using github PAT from envvar GITHUB_PAT
Downloading GitHub repo easystats/modelbased@HEAD
✓  checking for file ‘/private/var/folders/_p/55p16nb13cs7yy707d8l_v1s_rmpr4/T/Rtmp38TDiH/remotes2a7fa6eb8b/easystats-modelbased-7afc38f/DESCRIPTION’ ...
─  preparing ‘modelbased’:
✓  checking DESCRIPTION meta-information ...
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
   Removed empty directory ‘modelbased/docs’
   Removed empty directory ‘modelbased/pkgdown’
   Removed empty directory ‘modelbased/vignettes’
─  building ‘modelbased_0.6.0.tar.gz’
   Warning: invalid uid value replaced by that for user 'nobody'
   Warning: invalid gid value replaced by that for user 'nobody'
   
Installing package into ‘/Users/patil/Library/R/4.0/library’
(as ‘lib’ is unspecified)
* installing *source* package ‘modelbased’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error: object ‘parameters_table’ is not exported by 'namespace:insight'
Execution halted
```

**Comment by DominiqueMakowski** [comment]
will be fixed by the big #101

**Comment by IndrajeetPatil** [comment]
Cool. Closing in favor of #101
